python/cuml/svm/svm_base.pyx

```diff
@@ -537,21 +541,27 @@ class SVMBase(Base):
         state = self.__dict__.copy()
         del state['handle']
         del state['_model']
-        state['dual_coef_'] = cudf.DataFrame.from_gpu_matrix(self.dual_coef_)
-        state['support_'] = cudf.Series(self.support_)
-        state['support_vectors_'] = \
-            cudf.DataFrame.from_gpu_matrix(self.support_vectors_)
-        state['_unique_labels'] = cudf.Series(self._unique_labels)
 
+        # Only when the model is fit once we need to store these parameters
+        if self.fit_status_ == 0:
```

Comment:
yeah, I would assume that fit_status_ == 0 means we have _not_ fit it yet... I think boolean would be clearer.

Reply:
I agree that boolean would be clearer, but from what @tfeher mentioned; it looks like 0 for correct fit and 1 for solution not converging seems to be the standard that sklearn uses. 

Do you still want to change it to a boolean flag even though it doesn't comply with the method that sklearn is using?

Reply:
Oh, I see that in their docs now... ugh, not my favorite interface, but I think it's good to preserve compatibility so we can leave it like this. Thanks!